extension-app/src/components/Todo.jsx

```diff
@@ -1,7 +1,74 @@
-import React from 'react'
+import { useEffect, useState } from "react";
+import { FiTrash2 } from "react-icons/fi";
+import { v4 as uuid } from "uuid";
+import { useLocalStorage } from "../utils/hooks/useLocalStorage";
+
+const Todo = () => {
+  const [localTodos, setLocalTodos] = useLocalStorage("todos", []);
+  const [localCompTodos, setLocalCompTodos] = useLocalStorage("compTodos", []);
+
+  const [showTodo, setShowTodo] = useState(false);
+
+  const inputHandler = (event) => {
+    if (event.key === "Enter") {
+      setLocalTodos([
+        ...localTodos,
+        { _id: uuid(), task: event.target.value, isDone: false },
+      ]);
+      event.target.value = "";
+    }
+  };
+  const taskCompletionHandler = (taskObj) => {
+    setLocalCompTodos([...localCompTodos, taskObj]);
+    setLocalTodos(localTodos.filter((item) => item._id !== taskObj._id));
+  };
+  const deletionHandler = (id) => {
+    setLocalCompTodos(localCompTodos.filter((item) => item._id !== id));
+  };
 
-export const Todo = () => {
   return (
-    <div>Todo</div>
-  )
-}
+    <div className="todo-wrapper">
+      {showTodo && (
+        <div className="todo-container slide-left text-center px-s py-m br-m">
+          <h3 className="text-light">Today's tasks</h3>
+          <input
+            className="todo-input my-s px-s py-xs"
+            onKeyDown={inputHandler}
+            placeholder="Add a task and press Enter"
+          />
+          <ul className=" text-light text-left flex flex-col-reverse">
+            {localTodos.map((taskObj) => (
+              <li className="list-item flex justify-content-space-bet">
+                <span>{taskObj.task}</span>
+                <input
+                  type="checkbox"
+                  checked={taskObj.isDone}
+                  value={taskObj.task}
```

Comment:
You can destructure isDone and task from taskObj

Reply:
Thank you for pointing out will keep that in mind.